lib/new_relic/agent/instrumentation/logger.rb

```diff
@@ -16,9 +16,7 @@
 
   executes do
     ::NewRelic::Agent.logger.info "Installing Logger instrumentation"
-  end
 
-  executes do
```

Comment:
I saw you combined these 2 execute blocks like we were discussing above, but not on the other file active_support_logger.  Did you want to do it in both places?

Reply:
Great point! I forgot about that file. I'll make the fix :)